robotbase/src/main/java/frc/robot/subsystems/EndAffectorSubsystem.java

```diff
@@ -0,0 +1,49 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.Constants.CAN_ID;
+import frc.robot.Constants.END_AFFECTOR;
+
+public class EndAffectorSubsystem extends SubsystemBase {
+
+  private CANSparkMax m_motor;
+
+  public EndAffectorSubsystem() {
+    m_motor = new CANSparkMax(CAN_ID.END_AFFECTOR_MOTOR_ID, MotorType.kBrushed);
+    configure();
+  }
+
+  private void configure(){
+    m_motor.setInverted(END_AFFECTOR.IS_INVERTED);
+    m_motor.setIdleMode(END_AFFECTOR.IDLE_MODE);
+    m_motor.setSmartCurrentLimit(END_AFFECTOR.MOTOR_STALL_LIMIT_AMPS, END_AFFECTOR.MOTOR_FREE_LIMIT_AMPS);
+    m_motor.enableVoltageCompensation(12);
+  }
+
+  public void setPivotAxisSpeed(double axisSpeed) {
+    double motorSpeed = (-axisSpeed) * END_AFFECTOR.AXIS_MAX_SPEED;
+    m_motor.set(motorSpeed);
+  }
+
+  public void setSpeed(double speedPercentage) {
```

Comment:
Let's rename this to setAxisSpeed to be consistent with other subsystems.

Reply:
yeah i forgot to rename that. woops.